tests/cpu.rs

```diff
@@ -51,3 +51,14 @@ fn test_too_rapid_cpu_refresh() {
 
     assert!(s.cpus().iter().any(|c| !c.cpu_usage().is_nan()));
 }
+
+#[test]
+fn test_cpu_realtime_freq() {
```

Comment:
This test is not really useful in its current state: we never know if it retrieves the information, it just displays whatever it gets. You need to ensure that it returns `Some` on windows.